tools/ua.bash

```diff
@@ -1,25 +1,35 @@
 # bash completion for ubuntu-advantage-tools
 
+. /etc/os-release  # For VERSION_ID
+
 _ua_complete()
 {
     local cur_word prev_word services subcmds base_params
     cur_word="${COMP_WORDS[COMP_CWORD]}"
     prev_word="${COMP_WORDS[COMP_CWORD-1]}"
 
-    services=$(python3 -c "from uaclient.entitlements import valid_services; from uaclient.config import UAConfig; print(*valid_services(cfg=UAConfig()), sep=' ')
-")
-    subcmds=$(pro --help | awk '/^\s*$|^\s{5,}|Available|Use/ {next;} /Flags:/{check=1;next}/Use ubuntu-avantage/{check=0}check{ if ( $1 ~ /,/ ) { print $2} else print $1}')
+    if [ "$VERSION_ID" = "16.04" ] || [ "$VERSION_ID" == "18.04" ]; then
+        services="cc-eal cis esm-apps esm-infra fips fips-updates livepatch realtime-kernel ros ros-updates"
```

Comment:
```suggestion
        services="cc-eal esm-apps esm-infra fips fips-updates livepatch realtime-kernel ros ros-updates cis"
```
I know this isn't alphabetical, but then we wouldn't need so much repetition in our tests - thoughts? @renanrodrigo @lucasmoura ?

Reply:
imho alphabetical > test repetition